app/ui-react/packages/ui/src/Integration/IntegrationDetailHistoryListView.tsx

```diff
@@ -46,7 +47,7 @@ export class IntegrationDetailHistoryListView extends React.Component<
           </Grid.Row>
         ) : null}
 
-        {this.props.children ? (
+        {this.props.children && this.props.hasHistory === true ? (
```

Comment:
You could just pass the whole child element here and not bother having a `hasHistory`, then in the parent component do:
```
{ hasHistory && <Grid.Row blah blah blah
```
like [this](https://github.com/syndesisio/syndesis-react/blob/master/app/ui-react/packages/ui/stories/Integration/CiCd/CiCdManagePageUI.stories.tsx#L179) kinda thing